### PR TITLE
Add: Created Badge Component

### PR DESCRIPTION
**Pull request by developer-anand**
Badge Component

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -14,15 +14,94 @@
...
+  height: 8rem;
+  position: relative;
+  margin: auto;
+  width: 90%;
+}
+
+.icon-badge {
+  position: relative;
+  right: 0px;
+}
```
Comment (zoef-yt) [inline components/badge/badge.css:36]: Spacing

File: components/badge/badge.css [hunk 2]
```diff
@@ -14,15 +14,94 @@
...
+  height: 1rem;
+  width: 1rem;
+  border-radius: 50%;
+  left: 3rem;
+  top: 3rem;
+}
+
+.badge-online {
+  background-color: green;
+}
```
Comment (zoef-yt) [inline components/badge/badge.css:123]: spacing

File: components/badge/badge.css [hunk 3]
```diff
@@ -14,15 +14,94 @@
...
+.badge-inactive {
+  background-color: rgb(128, 19, 0);
```
Comment (zoef-yt) [inline components/badge/badge.css]: use var here to stay consistent

File: components/badge/badge.css [hunk 4]
```diff
@@ -14,15 +14,94 @@
...
+}
+
+.badge-offline {
+  background-color: grey;
```
Comment (zoef-yt) [inline components/badge/badge.css]: use var

File: components/badge/badge.css [hunk 5]
```diff
@@ -14,15 +14,94 @@
...
+
+.bg-blue {
+  background-color: blue;
+  border-radius: 50%;
+}
+
+.bg-pink {
+  background-color: pink;
+  border-radius: 50%;
+}
```
Comment (zoef-yt) [inline components/badge/badge.css]: Use var for color

File: documentation.css [hunk 6]
```diff
@@ -14,6 +14,10 @@
   display: flex;
   flex-direction: row;
 }
+
+.comp-list {
+  position: fixed;
+}
```
Comment (zoef-yt) [inline documentation.css:20]: spacing

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Comment by DivyanshSareen** [comment]
image
the height and width must be equal or you will get an ellipse

this might help make the badge circular

https://stackoverflow.com/questions/42165026/border-radius-generates-eclipse-need-90degree-rounded-corners-css

File: components/badge/badge.css (hunk 1 above)
Comment (developer-anand) [reply components/badge/badge.css:36]: done

File: components/badge/badge.css (hunk 4 above)
Comment (developer-anand) [reply components/badge/badge.css]: please check I have changed to var

File: components/badge/badge.css (hunk 5 above)
Comment (developer-anand) [reply components/badge/badge.css]: added

**Comment by archit7448** [comment]
makes badges notification circular and all other thing look good .

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Comment by developer-anand** [comment]
> image
> 
> the height and width must be equal or you will get an ellipse
> this might help make the badge circular
> 
> https://stackoverflow.com/questions/42165026/border-radius-generates-eclipse-need-90degree-rounded-corners-css

image

Please check now

**Review by Omkar-Ghate — COMMENTED** [review]
Looks good! Maybe you could show badges with different-sized avatars, just a suggestion.

**Review by Omkar-Ghate — APPROVED** [review]
Good to merge.

**Comment by developer-anand** [comment]
> Looks good! Maybe you could show badges with different-sized avatars, just a suggestion.

Thanks I will try it.

**Pull request merged**